server/controllers/getProduct.js

```diff
@@ -0,0 +1,10 @@
+
+const getProduct = (req,res)=>{
+    const {productId}=req.params;
```

Comment:
Validate this to check it's a number, you can do a simple validation by just doing this
```js
if (productId > 0) {
...
} else {
res.status(400).json({err...})
}
```